dockerfiles/musl-ci-linux/Dockerfile

```diff
@@ -0,0 +1,120 @@
+ARG VCS_REF=master
+ARG BUILD_DATE=""
+ARG REGISTRY_PATH=paritytech
+
+FROM ${REGISTRY_PATH}/base-ci-linux:latest
+
+# metadata
+LABEL io.parity.image.authors="devops-team@parity.io" \
+    io.parity.image.vendor="Parity Technologies" \
+    io.parity.image.title="${REGISTRY_PATH}/musl-ci-linux" \
+    io.parity.image.description="Inherits from base-ci-linux:latest. \
+rust target x86_64-unknown-linux-musl, openssl, zlib, musl tools" \
+    io.parity.image.source="https://github.com/paritytech/scripts/blob/${VCS_REF}/\
+dockerfiles/musl-linux/Dockerfile" \
+    io.parity.image.documentation="https://github.com/paritytech/scripts/blob/${VCS_REF}/\
+dockerfiles/musl-ci-linux/README.md" \
+    io.parity.image.revision="${VCS_REF}" \
+    io.parity.image.created="${BUILD_DATE}"
+
+ARG TARGET=x86_64-linux-musl
+
+# install tools and dependencies
+RUN rustup target add x86_64-unknown-linux-musl; \
+    rustup target add x86_64-unknown-linux-musl --toolchain nightly
+RUN set -eux; \
+    apt -y update; \
+    apt install -y --no-install-recommends \
+    gpg dirmngr gpg-agent;
+
+# config for x86_64-unknown-linux-musl
+COPY utility/musl-ci-linux-config /root/.cargo/config
+
+# install musl
+RUN set -eux;\
+    curl -fSL "https://more.musl.cc/x86_64-linux-musl/x86_64-linux-musl-cross.tgz" -o x86_64-linux-musl-cross.tgz; \
+    curl -fSL "https://more.musl.cc/x86_64-linux-musl/x86_64-linux-musl-cross.tgz.sig" -o x86_64-linux-musl-cross.tgz.sig; \
+    export GNUPGHOME="$(mktemp -d)"; \
+    gpg --batch --keyserver ha.pool.sks-keyservers.net --recv-keys BE4BF7E6811C5BA41345C11EB1D0B4566FBBDB40; \
```

Comment:
https://github.com/docker-library/official-images#security Best

Reply:
Well, OK then.
I would still prefer vendoring [the key file itself](https://keys.openpgp.org/vks/v1/by-fingerprint/BE4BF7E6811C5BA41345C11EB1D0B4566FBBDB40), but whatever.
We can always fix this when it will fail to fetch the keys the next time.